requirements.txt

```
@@ -1 +1 @@
-requests>=2.18.0,<2.18.99
+requests>=2.20.0
```

We shouldn't auto install/update major releases for the package.
```suggestion
requests>=2.20.0,<3.0.0
```